src/objects.h

```diff
@@ -6841,6 +6841,10 @@ class Script: public Struct {
   V(SIMD.float32x4, fromInt32x4, Int32x4ToFloat32x4, Float32x4, Int32x4)       \
   V(SIMD.float32x4, fromInt32x4Bits, Int32x4BitsToFloat32x4, Float32x4,        \
     Int32x4)                                                                   \
+  V(SIMD.float32x4, fromFloat64x2, Float64x2ToFloat32x4, Float32x4,            \
+    Float64x2)                                                                 \
+  V(SIMD.float32x4, fromFloat64x2Bits, Float64x2BitsToFloat32x4, Float32x4,    \
+    Float64x2)                                                                 \
```

Comment:
No need for the runtime implementation. Please put this into your CrankShaft PR.